README.md

```diff
@@ -1467,3 +1468,13 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
+ To assist with getting onboarded to the project, review the following documents.
+
+- [Azul Overview](#1-architecture-overview)  
+- [Contributing Guidlines](CONTRIBUTING.rst)
+- [Getting Started](#2-getting-started)
```

Comment:
I would reorder this so that `Getting Started` comes before `Contributing Guidelines`. Also Guidelines is spelled incorrectly here. If it's spelled incorrectly elsewhere can you correct it (ideally in a separate commit that comes before this one)?